### PR TITLE
Fix Guacamole shared drive always enabled

### PR DESCRIPTION
**Pull request by marrobi**
Related to #3885

Update Guacamole shared drive configuration to respect the 'drive' option setting.

* Change the default value of `guac_enable_drive` to `false` in `templates/workspace_services/guacamole/porter.yaml`.
* Change the default value of `guac_enable_drive` to `false` in `templates/workspace_services/guacamole/template_schema.json`.

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/microsoft/AzureTRE/issues/3885?shareId=342a8507-4362-486e-a3d3-413a5171a307).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ ENHANCEMENTS:
 BUG FIXES:
 * Fix numbering issue within `bug_report.md` template ([#4028](https://github.com/microsoft/AzureTRE/pull/4028))
 * Disable public network access to the API App Service ([#3986](https://github.com/microsoft/AzureTRE/issues/3986))
+* Fix Guacamole shared drive always enabled ([#4046](https://github.com/microsoft/AzureTRE/pull/4046))
```
Comment (tim-allen-ck) [inline CHANGELOG.md:17]: wrong issue url?

**Comment by tim-allen-ck** [comment]
I have a feeling this was more to do with even if you toggle the variable it doesn't change value? @Danny-Cooke-CK can you confirm?

File: CHANGELOG.md (hunk 1 above)
Comment (marrobi) [reply CHANGELOG.md:17]:
```suggestion
* Fix Guacamole shared drive always enabled ([#3885](https://github.com/microsoft/AzureTRE/issues/3885))
```

This better?

I'm thinking now GitHub will auto generate release notes (that we can amend), do we still need a changelog? Separate disucssion!

**Review by tim-allen-ck — APPROVED** [review]
LGTM, if this fixes Danny's issue

**Pull request merged**